src/Hl7.Fhir.Specification/Specification/StructureDefinitionSummaryProvider.cs

```diff
@@ -231,10 +239,11 @@ public XmlRepresentation Representation
         private static bool isResource(ElementDefinition defn) => defn.Type.Count == 1 && 
             (defn.Type[0].Code == "Resource" || defn.Type[0].Code == "DomainResource");
 
+        public string DefaultTypeName => _definition.GetExtensionValue<FhirString>("http://hl7.org/fhir/StructureDefinition/elementdefinition-defaulttype")?.Value;
```

Comment:
public string DefaultTypeName => _definition.GetStringExtension("http://hl7.org/fhir/StructureDefinition/elementdefinition-defaulttype");

Reply:
Heel klein dingetje, maar dit is iets netter denk ik..